routers/api/packages/container/container.go

```diff
@@ -143,6 +144,22 @@ func Authenticate(ctx *context.Context) {
 	u := ctx.Doer
 	if u == nil {
 		u = user_model.NewGhostUser()
+	} else if u.IsActions() {
+		task, err := actions_model.GetTaskByID(ctx, ctx.Data["ActionsTaskID"].(int64))
+		if err != nil {
+			apiError(ctx, http.StatusInternalServerError, err)
+		}
+		if err := task.LoadJob(ctx); err != nil {
+			apiError(ctx, http.StatusInternalServerError, err)
+		}
+		if err := task.Job.LoadRun(ctx); err != nil {
+			apiError(ctx, http.StatusInternalServerError, err)
+		}
+		if err := task.Job.Run.LoadTriggerUser(ctx); err != nil {
+			apiError(ctx, http.StatusInternalServerError, err)
+		}
```

Comment:
Is that correct?
If there's an error and the task is nil, and `task.Job` will panic, right?